### PR TITLE
Improve readme files

### PR DESCRIPTION
**Pull request by joneff**
**README**
* clarify which version of bootstrap we use
* move license section at the very end
  * touch 2020 to 2021 (can we do something about it?)
* merge getting started and usage sections
* add new section about customizing
  * extra info for material theme
* add new section about integrating with third party frameworks
  * provide example why it cannot be expected `form-control` to work properly on Kendo UI components
* add new section about bug reports and feature requests
* add new section about versioning

**package.json**
* add peer dependency to default theme
* add homepage field
* add bugs field

### CODE REVIEW

File: packages/bootstrap/README.md [hunk 1]
```diff
@@ -1,50 +1,136 @@
...
+
+You can customize Kendo UI Bootstrap Theme both in terms of what gets compiled and how the theme will look.
+
+### Importing
+
+In your custom csss file, you can import the entirety of the theme, by importing `dist/all.scss` or pick just the styles for the components you need. The files for individual components are in `scss/` folder:
+
+```scss
+// Import only PanelBar and Grid styles
+@import "~@progress/kendo-theme-default/scss/panelbar/_index.scss";
```
Comment (silviyaboteva) [inline packages/bootstrap/README.md:53]: kendo-theme-bootstrap

File: packages/bootstrap/README.md [hunk 2]
```diff
@@ -1,50 +1,136 @@
...
+@import "~@progress/kendo-theme-default/scss/grid/_index.scss";
```
Comment (silviyaboteva) [inline packages/bootstrap/README.md:54]: kendo-theme-bootstrap

File: packages/classic/README.md [hunk 3]
```diff
@@ -2,8 +2,119 @@
...
+
+You can customize Kendo UI Classic Theme both in terms of what gets compiled and how the theme will look.
+
+### Importing
+
+In your custom csss file, you can import the entirety of the theme, by importing `dist/all.scss` or pick just the styles for the components you need. The files for individual components are in `scss/` folder:
+
+```scss
+// Import only PanelBar and Grid styles
+@import "~@progress/kendo-theme-default/scss/panelbar/_index.scss";
```
Comment (silviyaboteva) [inline packages/classic/README.md:53]: kendo-theme-classic

File: packages/classic/README.md [hunk 4]
```diff
@@ -2,8 +2,119 @@
...
+@import "~@progress/kendo-theme-default/scss/grid/_index.scss";
```
Comment (silviyaboteva) [inline packages/classic/README.md:54]: kendo-theme-classic

File: packages/bootstrap/README.md [hunk 5]
```diff
@@ -1,50 +1,136 @@
...
+We've ensured that our themes work correctly with both those styles being present and absent.
+
+However, even when being cautious, neglecting the scope and cascade of CSS, that may result in unexpected side effects. Oddly enough, most of the cases when that happens, everything is expected, at least from CSS point of view, but it can be still frustrating.
+
+For instance, it cannot be expected that adding `form-control` from Bootstrap to a Kendo UI Combobox will make the component behave and look like a Bootstrap form component. The scope and cascade are different -- `form-control` is intended to be added to pure `input` elements, where as a Kendo UI Combobox has a different structure and styles will not be applied correctly.
+
+Another example is adding `bg-red-400` from Tailwind to a Kendo UI Button, depending on the exact stylesheet order that class may or may not be applied.
+
+As a general rule, it should not be expected that mixing at matching classes from different frameworks will yield the desired results.
+
```
Comment (silviyaboteva) [inline packages/bootstrap/README.md:111]: Can we add something about our swatches for the Bootstrap theme?

File: packages/classic/package.json [hunk 6]
```diff
@@ -45,6 +49,9 @@
     "prepublishOnly": "./build/embed-dependencies '@progress/kendo-theme-default'",
     "postpublish": "rm -rf modules && git checkout scss"
   },
+  "peerDependencies": {
+    "@progress/kendo-theme-default": "^4.41.0"
+  },
   "devDependencies": {
     "@progress/kendo-theme-default": "^4.41.0"
```
Comment (silviyaboteva) [inline packages/classic/package.json:56]: if the Default theme is in the peer dependencies list, do we need it in dev dependencies as well?

File: packages/default/README.md [hunk 7]
```diff
@@ -2,72 +2,133 @@
...
 
-To change the layout or the front meter of the generated help topic, change the [`build/customization.md.hbs`](/build/customization.md.hbs) source file.
+However, even when being cautious, neglecting the scope and cascade of CSS, that may result in unexpected side effects. Oddly enough, most of the cases when that happens, everything is expected, at least from CSS point of view, but it can be still frustrating.
+
+For instance, it cannot be expected that adding `form-control` from Bootstrap to a Kendo UI Combobox will make the component behave and look like a Bootstrap form component. The scope and cascade are different -- `form-control` is intended to be added to pure `input` elements, where as a Kendo UI Combobox has a different structure and styles will not be applied correctly.
+
+Another example is adding `bg-red-400` from Tailwind to a Kendo UI Button, depending on the exact stylesheet order that class may or may not be applied.
+
+As a general rule, it should not be expected that mixing at matching classes from different frameworks will yield the desired results.
+## Bug reports and feature requests
```
Comment (silviyaboteva) [inline packages/default/README.md:111]: Add one empty line before this heading

File: packages/default/README.md [hunk 8]
```diff
@@ -2,72 +2,133 @@
...
+
+Have a bug or a feature request? Please first search for existing and closed issues. If your problem or idea is not addressed yet, please open a new [bug report](https://github.com/telerik/kendo-themes/issues/new?labels=bug,T:Default&template=bug_report.md) or [feature request](https://github.com/telerik/kendo-themes/issues/new?labels=Enhancement,T:Default&template=feature_request.md).
+
+## Versioning
+
+To ensure version predictability, we use [Semantic Versioning](https://semver.org/): we try to add only feaures for minor versions and bugfixes (or smaller features) for patch version.
+## License
```
Comment (silviyaboteva) [inline packages/default/README.md:118]: add one empty line before this heading

File: packages/material/README.md [hunk 9]
```diff
@@ -2,9 +2,131 @@
...
+
+You can customize Kendo UI Material Theme both in terms of what gets compiled and how the theme will look.
+
+### Importing
+
+In your custom csss file, you can import the entirety of the theme, by importing `dist/all.scss` or pick just the styles for the components you need. The files for individual components are in `scss/` folder:
+
+```scss
+// Import only PanelBar and Grid styles
+@import "~@progress/kendo-theme-default/scss/panelbar/_index.scss";
```
Comment (silviyaboteva) [inline packages/material/README.md:53]: kendo-theme-material

File: packages/material/README.md [hunk 10]
```diff
@@ -2,9 +2,131 @@
...
+@import "~@progress/kendo-theme-default/scss/grid/_index.scss";
```
Comment (silviyaboteva) [inline packages/material/README.md:54]: kendo-theme-material

File: packages/material/README.md [hunk 11]
```diff
@@ -2,9 +2,131 @@
...
+We've ensured that our themes work correctly with both those styles being present and absent.
+
+However, even when being cautious, neglecting the scope and cascade of CSS, that may result in unexpected side effects. Oddly enough, most of the cases when that happens, everything is expected, at least from CSS point of view, but it can be still frustrating.
+
+For instance, it cannot be expected that adding `form-control` from Bootstrap to a Kendo UI Combobox will make the component behave and look like a Bootstrap form component. The scope and cascade are different -- `form-control` is intended to be added to pure `input` elements, where as a Kendo UI Combobox has a different structure and styles will not be applied correctly.
+
+Another example is adding `bg-red-400` from Tailwind to a Kendo UI Button, depending on the exact stylesheet order that class may or may not be applied.
+
+As a general rule, it should not be expected that mixing at matching classes from different frameworks will yield the desired results.
+## Bug reports and feature requests
```
Comment (silviyaboteva) [inline packages/material/README.md:123]: a new line before this heading

File: packages/material/README.md [hunk 12]
```diff
@@ -2,9 +2,131 @@
...
+
+Have a bug or a feature request? Please first search for existing and closed issues. If your problem or idea is not addressed yet, please open a new [bug report](https://github.com/telerik/kendo-themes/issues/new?labels=bug,T:Material&template=bug_report.md) or [feature request](https://github.com/telerik/kendo-themes/issues/new?labels=Enhancement,T:Material&template=feature_request.md).
+
+## Versioning
+
+To ensure version predictability, we use [Semantic Versioning](https://semver.org/): we try to add only feaures for minor versions and bugfixes (or smaller features) for patch version.
 ## License
```
Comment (silviyaboteva) [inline packages/material/README.md:130]: a new line before this heading

File: packages/nouvelle/README.md [hunk 13]
```diff
@@ -2,8 +2,120 @@
...
+
+You can customize Kendo UI Nouvelle Theme both in terms of what gets compiled and how the theme will look.
+
+### Importing
+
+In your custom csss file, you can import the entirety of the theme, by importing `dist/all.scss` or pick just the styles for the components you need. The files for individual components are in `scss/` folder:
+
+```scss
+// Import only PanelBar and Grid styles
+@import "~@progress/kendo-theme-default/scss/panelbar/_index.scss";
```
Comment (silviyaboteva) [inline packages/nouvelle/README.md:53]: kendo-theme-nouvelle

File: packages/nouvelle/README.md [hunk 14]
```diff
@@ -2,8 +2,120 @@
...
+@import "~@progress/kendo-theme-default/scss/grid/_index.scss";
```
Comment (silviyaboteva) [inline packages/nouvelle/README.md:54]: kendo-theme-nouvelle

File: packages/nouvelle/README.md [hunk 15]
```diff
@@ -2,8 +2,120 @@
...
+and then import it in your project styles:
+
+```scss
+// Import the entire theme
+@import "~@progress/kendo-theme-nouvelle/dist/all.scss";
+```
+
+For more information on how to implement the Nouvelle theme in your project, refer to the following articles:
+
+* [Using the Nouvelle Theme in Angular Projects](https://www.telerik.com/kendo-angular-ui/components/styling/theme-nouvelle/)
```
Comment (silviyaboteva) [inline packages/nouvelle/README.md:39]: We don't have such pages on this URL yet.

File: packages/classic/README.md [hunk 16]
```diff
@@ -2,8 +2,119 @@
...
+and then import it in your project styles:
+
+```scss
+// Import the entire theme
+@import "~@progress/kendo-theme-classic/dist/all.scss";
+```
+
+For more information on how to implement the Classic theme in your project, refer to the following articles:
+
+* [Using the Classic Theme in Angular Projects](https://www.telerik.com/kendo-angular-ui/components/styling/theme-classic/)
```
Comment (silviyaboteva) [inline packages/classic/README.md:39]: We don't have such pages on this URL yet.

File: packages/bootstrap/README.md [hunk 17]
```diff
@@ -1,50 +1,136 @@
...
+$input-bg: #f0f0f0;
+
+// Change just the autocomplete to have white background
+$autocomplete-bg: #ffffff;
+
+// Make the border of the grid black and slightly ticker
+$grid-border-width: 2px;
+$grid-border: #000000;
+
+// When you are finished with your customizatoni, import the theme
```
[the remark this answers is not in the document]
Comment (tsvetomir) [reply packages/bootstrap/README.md:78]: customizzazione 👨🏽 🇮🇹

File: packages/bootstrap/README.md (hunk 5 above)
Comment (joneff) [reply packages/bootstrap/README.md:111]: This is the readme. the one that's going to be shown in npm. it's more of an overview than a hyper detailed guide

File: packages/classic/package.json (hunk 6 above)
Comment (joneff) [reply packages/classic/package.json:56]: yes, we need it :)

File: packages/nouvelle/README.md (hunk 15 above)
Comment (joneff) [reply packages/nouvelle/README.md:39]: no, we dont, but I have to link to somewhere :D I am guessing, we should either have a generic page for that, or such pages for all themes. Also, we can track how many people click on the links

File: packages/classic/README.md (hunk 16 above)
Comment (joneff) [reply packages/classic/README.md:39]: see nouvelle note

**Review by JoomFX — APPROVED** [review]

**Review by silviyaboteva — APPROVED** [review]

File: packages/nouvelle/README.md (hunk 15 above)
Comment (silviyaboteva) [reply packages/nouvelle/README.md:39]: I'll go ahead and create such pages for the KendoReact, Kendo UI for Angular and Kendo UI for Vue docs sites: https://github.com/telerik/katana/pull/1098

**Pull request merged**